tests/homepage.spec.js

```diff
@@ -0,0 +1,22 @@
+import { preview } from "vite";
+import { test, expect } from "@playwright/test";
+
+test("front page has welcome text", async ({ page }) => {
+  await page.goto("/");
+  await expect(
+    page.getByRole("heading", { name: "Welcome to ATIP v1" })
+  ).toBeVisible();
+});
+
+test("choosing a local authority and clicking start changes the url", async ({
```

Comment:
This still uses a separate browser for each test. Per https://playwright.dev/docs/best-practices#write-fewer-tests-but-longer-tests, we could maybe fold some of these tests together later, especially once we get more and the runtime matters